relay-dynamic-config/src/feature.rs

```diff
@@ -40,6 +40,12 @@ pub enum Feature {
     /// Enable the Relay cardinality limiter.
     #[serde(rename = "organizations:relay-cardinality-limiter")]
     CardinalityLimiter,
+    /// Enable processing and extracting data from non dinamycally sampled profiles.
+    /// Only some data for slowest function aggregation will be used. The profile
+    /// itself won't be stored on GCS.
+    /// `<https://github.com/getsentry/snuba/blob/master/snuba/snuba_migrations/functions/0001_functions.py#L209-L256>`
```

Comment:
If we link to code and line numbers, we should link to a specific commit hash imo